lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,6 +1,58 @@
-const AWS = require('aws-sdk');
-exports.handler = async (event) => {
-    // TODO implement
-    const response = (success, reason, data) => { return { success: success, reason: reason, data: data } }
-    return response(true, null, null);
+const AWS = require("aws-sdk");
+const crypto = require("crypto");
+//AWS.config.loadFromPath("../../../keys.json");
+//Use the below code for local setup
+AWS.config.update({
+  region: "local",
+  endpoint: "http://localhost:8000",
+});
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: JSON.stringify(message),
+  };
+}
+
+function hmac_hash(secretKey, password) {
```

Comment:
don't you think using [bcrypt](https://www.npmjs.com/package/bcrypt) would remove these boilerplate codes??

Reply:
It would be. But then we have to install bcrypt module, just trying to minimise the number of of modules in lambda

Reply:
Lambda supports 10GB storage so it won't matter to add a couple of node modules, feel free to add them.
I suggest using bcrypt is more simpler and secure.

Reply:
bcrypt will make code much simpler and straightforward. Also, we have a 10GB storage limit for lambda so adding a couple of node modules won't matter much.